src/test/java/org/broadinstitute/consent/http/service/dao/DatasetServiceDAOTest.java

```diff
@@ -586,7 +586,7 @@ void testUpdateStudyWithPropUpdates() throws Exception {
     // Updated prop
     Optional<StudyProperty> updatedProp1 =
         updatedStudy.getProperties().stream()
-            .filter(p -> p.getStudyPropertyId().equals(prop1.getStudyPropertyId()))
+            .filter(p -> p.getKey().equals(prop1.getKey()))
```

Comment:
Since we're deleting and reinserting the property now, the ID can change.  Using the property Key ensures that we're finding the property we expect.